src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import React from 'react';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const getSelectedTab = () => (
+    tabs.find(tab => tab.id === selectedTabId) || tabs[0]
+  );
```

Comment:
you can have variable in App that contain whole selectedTab instead every time searching tab in tabs array by id